test/relations.test.js

```diff
@@ -2453,9 +2621,17 @@ describe('relations', function() {
       if (!author) return done();
       Author.findById(author.id, function(err, author) {
         author.pictures(function(err, pics) {
-          pics.should.have.length(2);
```

Comment:
Ouch, that's a good example of why it's a bad idea for one test to rely on the outcome of a previous test.

It would be best to rewrite this test to be independent, but I can also live with the current version as long as there is a comment making it clear why the test is asserting different results for different connectors.